composer.json

```diff
@@ -65,20 +65,24 @@
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
+    "extra": {
+        "enable-patching-note": "We need the following to allow rector/rector-src to patch dependencies.",
+        "enable-patching": true
+    },
```

Comment:
Kept getting errors about RectorConfig and a generator being null. Turns out that Rector patches symfony/dependency-injecting and other items. This is fine for `rector/rector` but not when using dev dependencies. I reviewed `rector/rector-phpunit` and found this line. By adding it, `symfony/dependency-injection is patched properly to make our tests run.

**NOTE** This will not affect Drupal sites or patch a Drupal site's `symfony/dependency-injection. This only happens for `require-dev`, when running our own PHPUnit tests. It has no effect on the root Composer project.